File: nhs_backend/java/src/main/java/com/bhtask/nhs/backend/BackendProperties.java
package com.bhtask.nhs.backend;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "com.bhtask.nhs.backend")
public class BackendProperties {

    private String dataPath;

    public String getDataPath() {
	return dataPath;
    }

    public void setDataPath(String dataPath) {
	this.dataPath = dataPath;
    }

}

File: nhs_backend/scala/src/main/scala/com/bhtask/nhs/backend/BackendProperties.scala
package com.bhtask.nhs.backend

import org.springframework.beans.factory.annotation.Value
import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties
class BackendProperties {

  @Value("${com.bhtask.nhs.backend.data-path}")
  var data: String = null

}
